pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -62,11 +65,17 @@ type TransformersConfig struct {
 	Feast TransformerConfig `json:"feast,omitempty"`
 }
 
+// +k8s:openapi-gen=false
+type CommonConfig struct {
+	QueueSideCarResourcePercentage string `json:"queueSideCarResourcePercentage"`
```

Comment:
Knative calls this the QueueProxy.

Reply:
NVM just saw: queue.sidecar.serving.knative.dev. Your name is great.

Reply:
I used the name from https://github.com/knative/serving/blob/ec457adcb1466abfcbc1c0a069e60285e769d135/pkg/apis/serving/register.go#L71 but I meant to remove this commonConfig because I no longer need expose this value from configmap. This line of code came back because of a mistake I made during my super long rebase :(